completions.go

```diff
@@ -56,9 +56,17 @@ func (c Commands) completionsFor(args []string) ([]string, shellcomp.Directive,
 		toComplete := args[0]
 		var name string
 
+		seen := make(map[string]bool)
```

Comment:
nit: you can use struct{} instead of bool here to save memory.

Reply:
True 😄 I saw that in _The Go Programming Language_ but I figured it costs 1 byte to use a `bool` (which is next-to-free) and it's a _tiny_ bit more legible